script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1.txt

```diff
@@ -87,7 +87,7 @@ The information that all the people that stayed at the hotel were engulfed in th
 <Page26>
 "――――"
 ...I try to hold back my nausea.
-I can't vomit here while thinking back to that tragedy.
+I can't vomit here while thinking back to that tragedy. //Maybe rephrase?
```

Comment:
> ここであの惨劇を思い返して嘔吐する事はできない
The essence is that he shouldn't try to remember what happened to prevent himself from vomiting.

I can't think back to the tragedy or I'll end up vomiting.